specification/app/resource-manager/Microsoft.App/stable/2025-01-01/CommonDefinitions.json

```diff
@@ -0,0 +1,1209 @@
+              },
+              "x-ms-identifiers": [
+                "code"
+              ]
+            },
+            "innererror": {
+              "description": "More information to debug error.",
+              "type": "string",
+              "readOnly": true
+            }
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "HttpScaleRule": {
+      "description": "Container App container Http scaling rule.",
+      "type": "object",
+      "properties": {
+        "metadata": {
+          "description": "Metadata properties to describe http scale rule.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "auth": {
+          "description": "Authentication secrets for the custom scale rule.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScaleRuleAuth"
+          },
+          "x-ms-identifiers": [
+            "triggerParameter"
+          ]
+        }
+      }
+    },
+    "TcpScaleRule": {
+      "description": "Container App container Tcp scaling rule.",
+      "type": "object",
+      "properties": {
+        "metadata": {
+          "description": "Metadata properties to describe tcp scale rule.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "auth": {
+          "description": "Authentication secrets for the tcp scale rule.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScaleRuleAuth"
+          },
+          "x-ms-identifiers": [
+            "triggerParameter"
+          ]
+        }
+      }
+    },
+    "QueueScaleRule": {
+      "description": "Container App container Azure Queue based scaling rule.",
+      "type": "object",
+      "properties": {
+        "queueName": {
+          "description": "Queue name.",
+          "type": "string"
+        },
+        "queueLength": {
+          "format": "int32",
+          "description": "Queue length.",
+          "type": "integer"
+        },
+        "auth": {
+          "description": "Authentication secrets for the queue scale rule.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScaleRuleAuth"
+          },
+          "x-ms-identifiers": [
+            "triggerParameter"
+          ]
+        }
+      }
+    },
+    "Scale": {
+      "description": "Container App scaling configurations.",
+      "type": "object",
+      "properties": {
+        "minReplicas": {
+          "format": "int32",
+          "description": "Optional. Minimum number of container replicas.",
+          "type": "integer"
+        },
+        "maxReplicas": {
+          "format": "int32",
+          "description": "Optional. Maximum number of container replicas. Defaults to 10 if not set.",
+          "type": "integer",
+          "default": 10
+        },
+        "cooldownPeriod": {
```

Comment:
>cooldownPeriod [](http://example.com/codeflow?start=9&length=14)

Description should mention the units of this value is seconds. I figured it out through logical inference using the default value, but should be explicitly stated.

Reply:
updated